1_OCaml/TP/TP5.ipynb

```diff
@@ -19,6 +19,136 @@
     "4. Tester si un tableau est trié par ordre croissant."
```

Comment:
<p>ça peut être bien d'utiliser une référence.</p><p>Ici, l'intérieur de la boucle for est de type bool, alors que CamL attend du unit</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>